filebeat/registrar/registrar.go

```diff
@@ -28,10 +28,10 @@ type Registrar struct {
 }
 
 var (
-	statesUpdate   = expvar.NewInt("registrar.states.update")
-	statesCleanup  = expvar.NewInt("registrar.states.cleanup")
-	statesCurrent  = expvar.NewInt("registrar.states.current")
-	registryWrites = expvar.NewInt("registrar.writes")
+	statesUpdate   = monitoring.NewInt(nil, "registrar.states.update")
```

Comment:
Looking at all the implementations now it seems like the name of the entry should probably be the first param (nothing to fix in this PR).